exporters/prometheus/src/testJpms/java/io/opentelemetry/exporters/prometheus/test/JpmsTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporters.prometheus.test;
+
+import io.opentelemetry.exporter.prometheus.PrometheusHttpServer;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import org.junit.jupiter.api.Test;
+
+class JpmsTest {
```

Comment:
I think the long term approach will be to add an extra CI configuration running Java 17 with modules for all our normal tests instead of defining jpms-specific test sets. Went with this as an interim and to verify the fix.